package.json

```diff
@@ -31,7 +31,7 @@
   "size-limit": [
     {
       "path": "dist/umd/index.js",
-      "limit": "25.0 KB"
```

Comment:
This is like inflation, always goes up, never goes down 😆

Reply:
by 3 bytes!